From b50f443639a6d6c36d5cd492d8f8da81b5beb5ce Mon Sep 17 00:00:00 2001
From: Junjie Mao <eternal.n08@gmail.com>
Date: Fri, 3 Apr 2015 09:58:18 +0800
Subject: [PATCH] Properly quote preprocessor command to ocamldep

Build in msys2 fails with not able to execute
''''camlp4\boot\camlp4boot.native' -D OPT' as the preprocessor for
ocamldep because the path is outside the quotes.

Signed-off-by: Junjie Mao <eternal.n08@gmail.com>
---
 myocamlbuild.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/myocamlbuild.ml b/myocamlbuild.ml
index 0573bef..d318c64 100644
--- a/myocamlbuild.ml
+++ b/myocamlbuild.ml
@@ -86,7 +86,7 @@ let () =
         let dep = "camlp4"/"boot"/exe in
         let cmd =
           let ( / ) = Filename.concat in
-          "camlp4"/"boot"/exe
+          "'" ^ "camlp4"/"boot"/exe ^ "'"
         in
         (Some dep, cmd)
     in
--
2.3.4
